<launch>

	<!-- Publisher for logged scans -->
	<param name="replay_path" value="/path/to/scan/diroctory/..." />
	<param name="replay_rate" value="0" />
	<node name="Publisher" pkg="slam3d_ros" type="velodyne_publisher" />

	<!-- Mapper -->
	<rosparam file="$(find slam3d_ros)/param/kitti_mapper.yaml"/>
	<node name="Mapper" pkg="slam3d_ros" type="mapper_node">
		<remap from="pointcloud" to="velodyne_scan" />
	</node>

	<!-- RVIZ to view the visualization -->
	<node name="RVIZ" pkg="rviz" type="rviz" args=" -d $(find slam3d_ros)/launch/kitti.rviz" />

</launch>

